### ISSUE TITLE
Email body gets cut

### ISSUE THREAD

**Issue by gvargas9** [nextcloud/mail#7879]
### Steps to reproduce

Email is shown in a window or frame. It is not possible to expand the email or see the full text.

### Expected behavior

Email should be shown in full

### Actual behavior

Email gets cut off
![Screenshot_20230108-111504_Chrome](https://user-images.githubusercontent.com/22826821/211207387-42494699-b29f-4085-a4d2-78dac7a26e57.jpg)

### PHP engine version

PHP 8.1

### Web server

Apache (supported)

### Database

MySQL

**Comment by GretaD** [comment]
Can you please add some reproducible steps, like for example your screen size and browser you are using, and if you're using mobile or web

**Comment by ChristophWurst** [comment]
Looks like the iframe resizer doesn't run. Compare the page URL with the one of the iframe that shows the message. Are they both using HTTP**S**?

**Comment by chicofreitas** [comment]
I looks like the iframe doesn't respond to dark theme.

**Comment by renueman** [comment]
I am confirming the issue. All the emails are being cut off in Nextcloud Mail at the bottom boarder it looks like.
I have tried changing themes but it does not make a difference. 
Apache2
Mariadb
Nextcloud version 25.0.3
Mail version 2.2.4
PHP 7.4
Debian 11
![email-cutoff2](https://user-images.githubusercontent.com/109969837/222972317-eb3fcebe-2783-4fe7-9668-c9ba7fb3921c.png)

**Comment by ChristophWurst** [comment]
Looks like the iframe resizer doesn't run. Compare the page URL with the one of the iframe that shows the message. Are they both using HTTPS?

**Comment by renueman** [comment]
> Looks like the iframe resizer doesn't run. Compare the page URL with the one of the iframe that shows the message. Are they both using HTTPS?

I finally figured the problem out. My understanding of the problem may not be correct but this is how I see it ---> The Reverse proxy is accepting connections on 443 ssl. Apache2 is behind the proxy and is just port 80 no ssl. Therefore some of the emails body is getting cut off because its not being transferred correctly to the reverse proxy. At first I thought it was probably some configuration issue with nginx (RP) or Apache2. Turns out there is a fix for this for the nextcloud config file:

`'overwriteprotocol' => 'https',`

I restarted apache2 for good measures and then email started showing the full body. I do not fully understand why except for my butchered discription above :) . However, It is now working fine. 

My simple layout:

WLAN -----> Nginx RP -------> Apache2 (Nextcloud)

I want to thank you for replying also. I appreciate it very much.
